src/pages/SudoPrompt.vue

```diff
@@ -35,17 +44,41 @@ import { ipcRenderer } from 'electron';
 import Vue from 'vue';
 import Checkbox from '@/components/form/Checkbox.vue';
 
+type SudoReason = 'networking' | 'docker-socket';
```

Comment:
This type declaration exists both here and `src/k8s-engine/lima.ts`. I'm wondering if there's a better place for this..

Reply:
I think it would be nice too, but I haven't found where it would go.  Once we figure out how to merge the translations, we'd move the text to the yaml files and it wouldn't be in here anymore anyway.